src/ppu.jakt

```diff
@@ -139,10 +139,7 @@ class PPU {
 
     function render_sprite_pixel(mut this, horizontal_pixel: u64, vertical_pixel: u64, is_behind: bool) throws -> bool {
         for sprite in 63..-1 {
-            let sprite_y = .oam[sprite * 4] as! u64
-            let sprite_index = .oam[sprite * 4 + 1] as! u8
-            let sprite_attr = .oam[sprite * 4 + 2] as! u8
-            let sprite_x = .oam[sprite * 4 + 3] as! u64
+            let sprite_attr = .oam[unchecked_add(unchecked_mul(sprite,4),2)]
```

Comment:
this is done for you automatically when you build in release mode, thankfully

Reply:
Still struggle with git sometimes, but I think I managed to amend this to remove the explicit unchecked adds.